pages.bn/common/alias.md

```diff
@@ -0,0 +1,29 @@
+# alias
+
+> উপনাম তৈরি করে -- এই শব্দগুলি যেগুলি কমান্ড স্ট্রিং দ্বারা প্রতিস্থাপন করা হয়।
+> উপনাম বর্তমান শেল সেশনে শেষ হয় যতক্ষণ না সেট করা হয়, উদাহরণ: `~/.bashrc`।
+> আরও জানতে: <https://tldp.org/LDP/abs/html/aliases.html>।
+
+- সমস্ত উপনামের তালিকা তৈরি করুন:
+
+`alias`
+
+- একটি সাধারণ উপনাম তৈরি করুন:
+
+`alias {{শব্দ}}="{{কমান্ড}}"`
+
+- দেওয়া উপনামে সংক্ষিপ্ত কমান্ড দেখুন:
+
+`alias {{শব্দ}}`
+
+- একটি উপনাম বাদ দিন:
+
+`unalias {{শব্দ}}`
+
+- `rm` কে একটি ইন্টারয়াক্টিভ কমান্ড হিসেবে পরিবর্তন করুন:
+
+`alias {{rm}}="{{rm -i}}"`
+
+- `ls -a` এর সংক্ষেপ রূপ হিসেবে `la` তৈরি করুন:
+
+`alias {{la}}="{{ls -a}}"`
```

Comment:
```suggestion
`alias {{la}}="{{ls -all}}"`
```
We prefer long options in tldr, this would also make this page align with the English page.